diff -u -u bnc2.9.3.orig/bncsetup bnc2.9.3/bncsetup
--- bnc2.9.3.orig/bncsetup	2004-10-09 23:54:49.000000000 +0300
+++ bnc2.9.3/bncsetup	2005-02-03 14:46:05.000000000 +0200
@@ -85,15 +85,15 @@
 
 }
 
-if ! [ -e ./mkpasswd ]
+if ! [ -e /usr/bin/bncmkpasswd ]
 then
   echo "You need to Make the package before you can use this script"
-  echo "(required ./mkpasswd)"
+  echo "(required /usr/bin/bncmkpasswd)"
   bsclean 0
 fi
 
 echo "BAD\$" > $TMPFILE
-echo "Test" | ./mkpasswd -s >> $TMPFILE
+echo "Test" | /usr/bin/bncmkpasswd -s >> $TMPFILE
 CRCYS=$(cat $TMPFILE | grep '\$' | tail -1 )
 
 
@@ -238,13 +238,13 @@
 
 if [ x$ERRORLEVEL = x0 ]
 then 
-  if [ -x ./mkpasswd ]
+  if [ -x /usr/bin/bncmkpasswd ]
   then
-    SUPERPASS="+"$(echo "$SUPERPASS" | ./mkpasswd -s)
-    BNCPASS="+"$(echo "$BNCPASS" | ./mkpasswd -s)
+    SUPERPASS="+"$(echo "$SUPERPASS" | /usr/bin/bncmkpasswd -s)
+    BNCPASS="+"$(echo "$BNCPASS" | /usr/bin/bncmkpasswd -s)
   else
     clear
-    echo "You have not did ./configure and make so there is no ./mkpasswd"
+    echo "You have not done ./configure and make so there is no /usr/bin/bncmkpasswd"
     echo "This program is required to produce the encryption"
     bsclean 0
   fi
