diff -urN bnc2.8.8.orig/sbuf.h bnc2.8.8/sbuf.h
--- bnc2.8.8.orig/sbuf.h	2002-07-11 05:17:56.000000000 +0200
+++ bnc2.8.8/sbuf.h	2003-08-09 21:59:15.000000000 +0200
@@ -35,4 +35,4 @@
 int sbuf_nextchunk(struct sbufstate *state);
 int sbuf_firstchunk(struct sbuf *record, struct sbufstate *state);
 int sbuf_getmsg(struct sbuf *record, char *buf, size_t length);
-int sbuf_gettag(struct sbuf *record, char *buf, size_t length);
\ No newline at end of file
+int sbuf_gettag(struct sbuf *record, char *buf, size_t length);
diff -urN bnc2.8.8.orig/server.c bnc2.8.8/server.c
--- bnc2.8.8.orig/server.c	2003-02-16 20:16:00.000000000 +0100
+++ bnc2.8.8/server.c	2003-08-09 21:59:32.000000000 +0200
@@ -31,7 +31,7 @@
 #include "struct.h"
 #include "send.h"
 #include "ctcp.h"
-
+#include "time.h"
 
 extern int h_errno;
 extern char logbuf[];
